securepy/stdcapture.py

```diff
@@ -64,20 +65,23 @@ def foo(*args, **kwargs):
     You can also use captured_std.stderr to obtain captured STDERR.
 
     If you don't want to lose STDOUT/STDERR captured values after function is done running,
-    you can specify `auto_reset=False` on init and run `StdCapture.reset` manually when needed.
+    you can specify `auto_reset=False` on init and run `IOCage.reset` manually when needed.
     You can also specify `memory_limit=100_000` in bytes (100kB) which will limit saved
     std storage size to that amount.
     """
 
-    def __init__(self, auto_reset: bool = True, memory_limit: int = 100_000):
+    def __init__(self, auto_reset: bool = True, memory_limit: int = 100_000, stdin=None):
```

Comment:
Please use typing to mark the type of this variable, for easier debugging and to mark what kind of variable this is to user
```suggestion
    def __init__(self, auto_reset: bool = True, memory_limit: int = 100_000, stdin: t.Optional[str] = None):
```